modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/rel/AbstractExecutionTest.java

```diff
@@ -100,14 +100,18 @@ protected ExecutionContext<T> executionContext(boolean withDelays) {
                     false,
                     0);
 
-            StripedThreadPoolExecutor stripedThreadPoolExecutor = (StripedThreadPoolExecutor) IgniteTestUtils.getFieldValue(
+            StripedThreadPoolExecutor stripedThreadPoolExecutor = IgniteTestUtils.getFieldValue(
                     taskExecutor,
                     QueryTaskExecutorImpl.class,
                     "stripedThreadPoolExecutor"
             );
-            stripedThreadPoolExecutor.shutdown();
 
-            IgniteTestUtils.setFieldValue(taskExecutor, "stripedThreadPoolExecutor", testExecutor);
+            // change it once on startup
+            if (!(stripedThreadPoolExecutor instanceof IgniteTestStripedThreadPoolExecutor)) {
```

Comment:
BTW, if the condition doesn't met, then stripedThreadPoolExecutor will not be stopped.